src/web/app/pages-instructor/student-profile/student-profile.ts

```diff
@@ -1,14 +0,0 @@
-import { Gender } from '../../../types/api-output';
-
-/**
- * Contains a student's profile
- */
-export interface StudentProfile {
-  shortName: string;
-  email: string;
-  institute: string;
-  nationality: string;
-  gender: Gender;
```

Comment:
I think you can also remove `Gender` type and use `Gender` in `api-output.ts`

Reply:
Oh I already removed the `Gender` type in one of my earlier PRs

https://github.com/TEAMMATES/teammates/pull/9909